ci-operator/step-registry/ipi/gcp/private/ipi-gcp-private-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
```

Comment:
If we already have ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/private/cucushift-installer-rehearse-gcp-ipi-private-workflow.yaml in this PR, we do not need this file here,